### ISSUE TITLE
Error while performing supervised classification

### ISSUE THREAD

**Issue by BlcaKHat** [spectralpython/spectral#81]
I am performing supervised classification .
In Gaussian Maximum Likelihood Classification , I am getting error in this step                                                    "  clmap = gmlc.classify_image(img)"
![classifiction](https://user-images.githubusercontent.com/16970999/38482689-f633491a-3bed-11e8-8f6c-7bcb4690f739.png)

**Comment by tboggs** [comment]
How many bands are in your image? It appears that you are undersampled, which is causing a singular covariance matrix. You need at least as many samples per class as there are bands in your image data.

**Comment by BlcaKHat** [comment]
I have 150 bands in my image. how can i fix it ?

**Comment by BlcaKHat** [comment]
I reduced the spectral subset to 13 bands and I also reduced the spatial subset., still i am getting the same error.
Is there a way to perform SAM classification with the help of saved roi or spectral library.

**Comment by BlcaKHat** [comment]
https://drive.google.com/open?id=1OyQSFSF_lrbvb6ss1GvLjXlCnwG3op4J

@tboggs  this is the data I am trying to classify. will you please give a hit . It would be really helpfull for me.

**Comment by BlcaKHat** [comment]
https://drive.google.com/open?id=1H28nGBOq-ywoh8x5DGqwSLZZxCIYDmC1
this is the sample data .It would be great if you give a hit.

**Comment by tboggs** [comment]
I don't know what you would expect me to do with those files. There is no ground truth mask to define training data labels.

The use of SAM and MSAM is described on the web site.
